src/core.c/Main.pm6

```diff
@@ -89,10 +89,10 @@ my sub RUN-MAIN(&main, $mainline, :$in-as-argsfiles) {
                 # implicit value
                 else {
                     if $bundling && nqp::iseq_s($0.Str, '-') {
-                        die "Can't combine bundling with explicit negation" if $1.chars;
+                        die "Can't combine bundling with explicit negation" if $1.chars && $arg.chars > 1;
```

Comment:
Wouldn't that make more sense as part of the preceding `if` condition?

Reply:
@Leont I don't think so. If I would add that to the enclosing `if`, it would fall back to interpreting the parameter as a non-bundled parameter as soon as more than one letter is passed even when it started with a single "-". So `-/abc` would be interpreted as `abc = False`. I'm pretty sure that's not what we want.